app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -20,6 +20,61 @@ def self.description
     @description ||= "Microsoft System Center VMM".freeze
   end
 
+  def self.params_for_create
+    @params_for_create ||= {
```

Comment:
No constant like the other providers?

Reply:
I thought a constant plus a class method could be confusing, didn't want people to use the constant instead.  Plus the rest of this class uses the `@ ||=` style to prevent allocating these on every method call

Reply:
Ok...I'd almost rather we change the other ones with constants to be consistent, then.